README.md

```diff
@@ -779,6 +779,9 @@ In both cases, it is necessary to tell the Node where the Event Bus is, so it ca
 the purpose of the `SE_EVENT_BUS_HOST`, `SE_EVENT_BUS_PUBLISH_PORT` and `SE_EVENT_BUS_SUBSCRIBE_PORT` environment
 variables.
 
+In some cases, for example if you want to tag a node, it might be necessary to supply a custom stereotype to the node config. The environement variable `SE_NODE_STEREOTYPE`
```

Comment:
Can you please add an example of this?

Reply:
For the docker command below?

Reply:
Yes, how one would start a container and use the `SE_NODE_STEREOTYPE` env var.

Reply:
Done